src/main/java/duke/exception/EmptyArgumentException.java

```diff
@@ -0,0 +1,17 @@
+package duke.exception;
+
+public class EmptyArgumentException extends DukeException {
+    private String arg;
+
+    public EmptyArgumentException() {
+        this.arg = "Arguments";
+    }
+    public EmptyArgumentException(String arg) {
+        this.arg = arg;
+    }
+
+    @Override
+    public String toString () {
+        return String.format("%s cant be empty bro", arg) ;
```

Comment:
Should the white space before the trailing semi-colon be removed? Noticed this in a couple of other areas as well.